lab2/two_drones_pkg/launch/two_drones.launch

```diff
@@ -1,11 +1,11 @@
 
 <launch>
-  <arg name="static" default="false" />
+  <arg name="static" default="false"/>
 
   <!-- Transform publishers !-->
   <group if="$(arg static)">
-    <node pkg="tf" type="static_transform_publisher" name="av1broadcaster" args="1 0 0 0 0 0 1 world av1 100" />
-    <node pkg="tf" type="static_transform_publisher" name="av2broadcaster" args="0 0 1 0 0 0 1 world av2 100" />
```

Comment:
Just curious: what was 100 for originally? Frequency?

Reply:
yeah, frequency (the original `tf` doesn't have the concept of a static transform so you have to publish them repeatedly)